components/expenses/ExpenseForm.js

```diff
@@ -570,6 +579,39 @@ const ExpenseFormBody = ({
                     {errors.payoutMethod.data.currency.toString()}
                   </Box>
                 )}
+                <StyledHr flex="1" borderColor="white.full" mx={2} />
+                {showModal ? (
+                  <ConfirmationModal
+                    show
+                    onClose={() => setShowModal(false)}
+                    header={formatMessage(msg.resetExpense)}
+                    body={formatMessage(msg.confirmResetExpense)}
+                    continueHandler={() => {
+                      setStep(STEPS.PAYEE);
+                      resetForm({ values: expense || getDefaultExpense(collective) });
```

Comment:
Doesn't formik automatically resets the form to its initial state if `values` is not provided?

Reply:
It does but in the case mentioned in https://github.com/opencollective/opencollective-frontend/pull/6958#discussion_r720758995 the initial state is edited expense and not the original expense.